### PR TITLE
change `ctrl+num` hotkeys to `alt+num`

### PR DESCRIPTION
**Pull request by firezym**
I have changed `ctrl+num` hotkeys to `alt+num`. It solves the `ctrl+num` conflicts on windows chrome.
I tested it locally. Looks working fine.
Now only need to press `alt+1  alt+2` or `alt +1 +2` to grid the cell half width, also easy to use.

### CODE REVIEW

**Comment by parmentelat** [comment]
hey; thanks for your contribution

reading this I just realized that the reason why I had picked the 'Ctrl-\` in the first place was precisely to protect against unforeseen conflicts

**digression**:
it was a slightly different context, but one day I had written an extension that redefined `Ctrl-X`, and then the first windows user shouted that it was stealing the global 'Cut' shortcut, something I failed to see because on a mac (and because I did not think much about it tbh)
so still in that other context I decided to prefix everything with `Ctrl-\`, and that turned out to work fine as far as `Ctrl-X` at least
and that's the historical reasoning behind that `Ctrl-\` prefix that we have in the current version, and that your PR is proposing to remove

in a nutshell since we use all five digits from 1 to 5, it may look safer to use a common prefix, as it limits the probability for conflicts

for example in my current setup I use Ctrl-1 to set the current cell in markdown with a level1 title..
I also have Opt-6 bound to some weird french-accent business..
I know...

point is the keyboard shortcut space is overcrowded !

so my proposal is to go for your PR for now, but I acknowledge the need for further work - I am thinking allowing for customization via settings; for example we could have the choice of using or not a configurable hotkey (like `Alt-\`)

how does that sound ?
